atomic_reactor/plugins/exit_remove_worker_metadata.py

```diff
@@ -0,0 +1,41 @@
+"""
+Copyright (c) 2017 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+from atomic_reactor.plugin import ExitPlugin
+from atomic_reactor.constants import PLUGIN_REMOVE_WORKER_METADATA_KEY
+from osbs.exceptions import OsbsResponseException
+
+__all__ = ('RemoveWorkerMetadataPlugin', )
```

Comment:
I'd just remove this as it's not really needed since we don't do `import *` anywhere. In which case, we'd probably want to include `defer_removal` in it.

Reply:
ok